media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -4405,13 +4405,16 @@ VAStatus DdiMedia_DeriveImage (
     vaimg->width                    = mediaSurface->iWidth;
     vaimg->height                   = mediaSurface->iRealHeight;
     vaimg->format.byte_order        = VA_LSB_FIRST;
+    GMM_RESOURCE_INFO          *gmmResourceInfo = mediaSurface->pGmmResourceInfo;
+    vaimg->data_size                = (uint32_t)gmmResourceInfo->GetSizeSurface();
+    uint32_t UPlaneXOffset          = (uint32_t)gmmResourceInfo->GetPlanarXOffset(GMM_PLANE_U);
+    uint32_t UPlaneYOffset          = (uint32_t)gmmResourceInfo->GetPlanarYOffset(GMM_PLANE_U);
```

Comment:
Suggestion: make these two vars `const`, so that readers can assume they are essentially aliases.

Reply:
agree